sanic_jwt/validators.py

```diff
@@ -37,11 +37,18 @@ def normalize(scope):
     return is_valid
 
 
-async def validate_scopes(request, scopes, user_scopes, require_all=True, require_all_actions=True, *args, **kwargs):
-    if asyncio.iscoroutinefunction(scopes):
-        scopes = await scopes(request, *args, **kwargs)
-    elif callable(scopes):
-        scopes = scopes(request, *args, **kwargs)
+async def validate_scopes(
+    request,
+    scopes,
+    user_scopes,
+    require_all=True,
+    require_all_actions=True,
+    *args,
+    **kwargs
+):
+    scopes = scopes(request, *args, **kwargs)
```

Comment:
This will result in a `TypeError` because scopes might also be a list or a string.

There are 4 valid parameters to pass for `scopes`:
1. string - `'customer:write'`
1. list - `['customer:write', 'customer:read']`
1. method
1. coroutine

Reply:
Hmmm, I missed that completely. Again, we need some unit test on that ... I will borrow from examples and create a test for it.

Reply:
No worries. If you can add the tests, great. If not, we can get them in the next pass.

Reply:
Yeap, working on them as we speak :wink:

Reply:
This really increased the coverage from 80 to 87%, using only the [provided example](https://github.com/ahopkins/sanic-jwt/blob/master/example/scopes.py) :wink: